src/Apprenticeships/SFA.DAS.Apprenticeships.Api/Controllers/ApprenticeshipController.cs

```diff
@@ -39,13 +45,16 @@ public async Task<ActionResult> GetApprenticeshipPrice(Guid apprenticeshipKey)
                 var employer = await _apiCommitmentsClient.Get<GetAccountLegalEntityResponse>(new GetAccountLegalEntityRequest(apprenticePriceInnerModel.AccountLegalEntityId.Value));
                 employerName = employer?.LegalEntityName;
             }
-            
-            var apprenticeshipPriceOuterModel = new ApprenticeshipPriceResponse
+
+            var calendarRange = await _collectionCalendarApiClient.Get<GetAcademicYearsResponse>(new GetAcademicYearsRequest(DateTime.Now));
```

Comment:
var calendarRange = await _collectionCalendarApiClient.Get<GetAcademicYearsResponse>(new GetAcademicYearsRequest(DateTime.Now.AddYears(-1)));
if(DateTime.Now > calendarRange.HardCloseDate)
{
     //earliest allowed date is 1st Aug current academic year
 }
 else
 {
     //earliest allowed date is 1st Aug previous academic year
 }